src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IForEachLoopStatement.cs

```diff
@@ -29,7 +29,9 @@ static void Main()
 }
 ";
             string expectedOperationTree = @"
-IForEachLoopStatement (Iteration variable: System.String value) (LoopKind.ForEach) (OperationKind.LoopStatement) (Syntax: 'foreach (st ... }')
+IForEachLoopStatement (LoopKind.ForEach) (OperationKind.LoopStatement) (Syntax: 'foreach (st ... }')
+  Locals: Local_1: System.String value
+  LoopControlVariable: ILocalReferenceExpression: value (IsDeclaration: True) (OperationKind.LocalReferenceExpression, Type: System.String, Constant: null) (Syntax: 'foreach (st ... }')
```

Comment:
>foreach (st ... } [](start = 166, length = 17)

It looks like syntax for this node is the entire foreach and I don't see a better candidate for it. This is bad. Perhaps the node should be marked as Implicit.

Reply:
Added a note to issue https://github.com/dotnet/roslyn/issues/22028#issuecomment-328499523 to discuss it.